CODEOWNERS

```
@@ -43,6 +43,9 @@
 # foreman_templates
 /packages/plugins/*foreman_templates/ @theforeman/packaging @theforeman/foreman_templates
 
+# katello
+/packages/katello/*katello/ @theforeman/packaging @theforeman/katello
```

As mentioned in the message, should this use the already existing katello release owners group?
```suggestion
/packages/katello/*katello/ @theforeman/packaging @theforeman/katello-release-owners
```